src/error.js

```diff
@@ -361,7 +367,14 @@ export function reportErrorToServerOrViewer(win, data) {
   return maybeReportErrorToViewer(win, data).then(reportedErrorToViewer => {
     if (!reportedErrorToViewer) {
       const xhr = new XMLHttpRequest();
-      xhr.open('POST', urls.errorReporting, true);
+      // Override the errorReportingUrl to test the new error reporting endpoint.
+      const newErrorReportingUrl =
+        'https://us-central1-amp-error-reporting.cloudfunctions.net/r';
+      const url =
+        IS_ESM && isExperimentOn(win, 'error-url')
```

Comment:
I don't think we should rely on experiment checking here, since this now depends on the cookie code to run. Why not just do a simple `Math.random()`, possibly with a `getMode` check to see if we're in canary, etc.

Reply:
sounds good to me, ill make the change